src/Support/GeometryProxy.php

```diff
@@ -163,11 +163,7 @@ private function determineCoordinateIndices($index, $length)
      */
     protected function exposeRawIfAvailable($argument)
     {
-        if (method_exists($argument, 'getRawGeometry')) {
-            return $argument->getRawGeometry();
-        }
-
-        return $argument;
+        return (method_exists($argument, 'getRawGeometry')) ? $argument->getRawGeometry() : $argument;
```

Comment:
Well, I have an answer, but it is not a great one...

The code coverage sees it as one line & there is not a good way to write the test to get the else with the way that the method_exist works.  This gives me 100% without having to tweak my test ;-)

I don't mind putting it back the other way, but I am not going to mess with he required changes to the test.

Just let me know what you prefer...